cocos/2d/CCDrawNode.h

```diff
@@ -313,12 +313,22 @@ class CC_DLL DrawNode : public Node
     
     // Overrides
     virtual void draw(Renderer *renderer, const Mat4 &transform, uint32_t flags) override;
+
+    void visit(Renderer* renderer, const Mat4 &parentTransform, uint32_t parentFlags) override;
     
     void setLineWidth(GLfloat lineWidth);
 
     // Get CocosStudio guide lines width.
     GLfloat getLineWidth();
 
+    /**
+    * When isolated is set, `visit()` will always get and identity as parent transform matrix.
+    * It's needed when DrawNode is used as absolute node without any effects from parent node.
+    */
+    void setIsolated(bool isolated) { _isolated = isolated; }
+
+    bool getIsolated() { return _isolated; }
```

Comment:
You must use the keyword `const` if the function is immutable. In this case, the function is immutable. Another good advantage would be when you reuse a macro `CC_SYNTHESIZE(bool, _isolated, Isolated)` or change the name of the getter from `bool getIsolated() const` to `bool isIsolated() const`